platform/mellanox/get_component_versions/get_component_versions.py

```diff
@@ -113,13 +128,23 @@ def main():
         return
 
     compiled_versions = parse_compiled_components_file()
-    platform_versions = get_platform_component_versions()
+    simx_compiled_ver = compiled_versions.pop("SIMX")
```

Comment:
@skr31 is this a typo?

Reply:
The SimX version will always appear in /etc/mlnx/component_versions since it's created during compilation and the image can be installed both on regular switch and SimX switch